src/main/java/com/gluonhq/substrate/util/windows/MSIBundler.java

```
@@ -0,0 +1,314 @@
+ * DISCLAIMED. IN NO EVENT SHALL GLUON BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.gluonhq.substrate.util.windows;
+
+import com.gluonhq.substrate.Constants;
+import com.gluonhq.substrate.model.InternalProjectConfiguration;
+import com.gluonhq.substrate.model.ProcessPaths;
+import com.gluonhq.substrate.model.ReleaseConfiguration;
+import com.gluonhq.substrate.util.FileOps;
+import com.gluonhq.substrate.util.Logger;
+import com.gluonhq.substrate.util.ProcessRunner;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Writer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.text.MessageFormat;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MSIBundler {
+
+    private final ProcessPaths paths;
+    private final InternalProjectConfiguration projectConfiguration;
+    private final String sourceOS;
+    private final Path rootPath;
+
+    public MSIBundler(ProcessPaths paths, InternalProjectConfiguration projectConfiguration) {
+        this.paths = paths;
+        this.projectConfiguration = projectConfiguration;
+        this.sourceOS = projectConfiguration.getTargetTriplet().getOs();
+        this.rootPath = paths.getSourcePath().resolve(sourceOS);
+    }
```

We should try to find WiX installer as soon as possible and fail fast if it is not found.
Also an indication on how to install it would be good.